components/speedreader/common/speedreader_panel.mojom

```diff
@@ -9,6 +9,10 @@ interface PanelFactory {
   CreateInterfaces(pending_receiver<PanelHandler> panel_handler, pending_receiver<PanelDataHandler> panel_data_handler);
 };
 
+interface UIHandler {
+  OnSiteSettingsChanged(SiteSettings settings);
```

Comment:
I can't find any reference to this function - is it being used?

Reply:
This is a left over. I'll remove it.